bin/critterget.py

```diff
@@ -801,13 +818,17 @@ def getCredentials(sessionKey):
 
         # return first set of credentials
         if (debug) : print "Entities is ", entities
-        if entities is not None:
+        if entities:
             for i, c in entities.items():
```

Comment:
You're working in this loop though, and it's pretty confusing.  Let's fix it please.